components/Toolbar/ToolbarLayout.js

```diff
@@ -1,8 +1,20 @@
 import React from "react";
-import { FormattedMessage } from "react-intl";
+import { defineMessages, FormattedMessage, intlShape, injectIntl } from "react-intl";
 import PropTypes from "prop-types";
 import { Filter, Toolbar } from "patternfly-react";
 
+const messages = defineMessages({
+  filterNameID: {
+    defaultMessage: "name"
+  },
+  filterReleaseID: {
+    defaultMessage: "release"
+  },
+  filterVersionID: {
+    defaultMessage: "version"
```

Comment:
One minor nit here regarding capitalization. Could we use upper-case for the beginning of each, i.e. "Name" instead of "name"?

Reply:
In master these are displayed as lower case. This is updated now.

Reply:
Thanks!!